src/Core/Validation/VerbValidation.fs

```diff
@@ -3,28 +3,23 @@
 open Article
 open Archaisms
 
-let tryGetVerb =
-    tryGetContent
-    >> Option.filter (hasChildPart "čeština")
-    >> Option.map (getChildPart "čeština")
-    >> Option.filter (hasChildPart "sloveso")
-    >> Option.map (getChildPart "sloveso")
-
 let hasRequiredInfoParticiple = 
-    tryGetVerb
-    >> Option.exists (hasChildPart "časování")
+    isMatch [
+        Is "čeština"
+        Is "sloveso"
+        Is "časování"
+    ]
 
 let hasRequiredInfoImperative = 
-    let hasImperative = 
-        Option.filter (hasChildPart "časování")
-        >> Option.map (getChildPart "časování")
-        >> Option.map getTables
-        >> Option.map (Seq.map fst)
-        >> Option.map (Seq.contains "Rozkazovací způsob")
-        >> Option.contains true
-
-    tryGetVerb
-    >> hasImperative
+    ``match`` [
+        Is "čeština"
```

Comment:
is it possible to not repeat čeština everywhere and make it like a mandatory match for everything?

Reply:
Hmm, actually this is a good idea. I thought to try to keep the framework general but we now hardcode the Czech wiki address there anyway... I will do that.

Reply:
So I added the function `getCzechContent` to the `Article` module that grabs the Czech part of the page and removed "čeština" from all the other places. Check it out!